File: carro.cpp
#include <iostream>
using namespace std;

class Carro {

private:
    int velocidade;

public:
    Carro() {
        velocidade = 1;
    }

    void acelerar(int taxa) {
        velocidade = velocidade * taxa;
    }

    int getVelocidade() {
        return velocidade;
    }
};

int main() {

    Carro meuCarro;

    meuCarro.acelerar(10);
    meuCarro.acelerar(2);

    cout << "Velocidade final: " << meuCarro.getVelocidade() << endl;

    return 0;
}

File: carro.py
class Carro:
    def __init__(self):
        self.velocidade = 1

    def acelerar(self, taxa):
        self.velocidade = self.velocidade * taxa

    def getVelocidade(self):
        return self.velocidade


def main():
    meuCarro = Carro()

    meuCarro.acelerar(10)
    meuCarro.acelerar(2)

    print("Velocidade final:", meuCarro.getVelocidade())


if __name__ == "__main__":
    main()
